### ISSUE TITLE
[Feature]

### ISSUE THREAD

**Issue by shreekrishnaacharya** [Udayraj123/OMRChecker#99]
Can we also add Optical character reader in this application for reading student names and other detail?

**Comment by Adwait01-Bhosale** [comment]
Hello, I can clearly contribute to this issue. As I have many a times worked upon tesseract OCR, Vision API OCR, and Microsoft OCR wherein I had made changes in the OCR by computer vision techniques like gray scaling the image and even giving it proper contrast to read the text accurately.

**Comment by shreekrishnaacharya** [comment]
Thank you Adwait01. i am also working on this OCR part and would appreciate your part of contribution. actually i am working on my thesis and i need to read document that consist of Mark and some text character. You contribution would be very helpful. Thank you

**Comment by Udayraj123** [comment]
Hi @Adwait01-Bhosale, that's great to hear! Can you and @shreekrishnaacharya join on over discord and discuss possible solutions with me?

**Comment by Udayraj123** [comment]
Here is a related issue for which I have framed a solution - https://github.com/Udayraj123/OMRChecker/issues/73

**Comment by shreekrishnaacharya** [comment]
https://github.com/Udayraj123/OMRChecker/issues/99#issuecomment-1328764312
@Udayraj123 , @Adwait01-Bhosale my discord account is **_sksharma72000_** . I have worked on tesseract OCR, it can detect printed text well however it is not suitable for handwriting detection.

**Comment by Udayraj123** [comment]
That's fine for a start. I have a requirement from a business regarding the same. You can help read printed roll numbers or booklet numbers in an area on the image. Later we can scale it for handwriting detection

**Comment by Udayraj123** [comment]
@shreekrishnaacharya @Adwait01-Bhosale let me know if you folks had any progress over this tasks. If so, others may decide to work on top of that.

**Comment by shreekrishnaacharya** [comment]
Yes i have update on it. And make use of it as well. But i dont know hot i
can share this task with you. Let me see that. Or please  reply the
instruction

**Comment by Udayraj123** [comment]
@shreekrishnaacharya you can create a PR with a running code on one of the samples(make sure to take latest master pull)

**Comment by JarEXE** [comment]
Hi I saw that #206 was merged last september. Is OCR functionality fully implemented?
